tests/Client/UpdaterTest.php

```diff
@@ -938,9 +912,9 @@ protected function assertClientFileVersions(array $expectedVersions): void
      * @param array $expectedUpdatedVersions
      *   The expected repo file version after refresh attempt.
      *
-     * @return void
-     *
      * @dataProvider providerExceptionForInvalidMetadata
+     *
+     * @testdox Invalid metadata in $fileToChange raises an exception
```

Comment:
We could make this even clearer (something like `Setting signed:version to 6 in root.json raises an exception`), but that is best done in a follow-up, since it would require me to refactor the test logic slightly.